### DISCUSSION TITLE
Welcome to gopro-dashboard-overlay Discussions!

### DISCUSSION THREAD

**Discussion by time4tea** [time4tea/gopro-dashboard-overlay#59] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  Please do let others know about tips for creating cool layouts, and how you use the software. 

*Screenshots are particularly welcome!*

Welcome to all! 

If you like the software **please** do toot, tweet, insta etc about it!

**Comment by aweussom** [comment]
Link to docs are broken?

https://pypi.org/project/gopro-overlay/docs/xml

**Reply by time4tea** [reply]
Hiya - ah - yes maybe - the links *do* work on the github pages - perhaps we need to use fully-qualified links so that they work on PyPI. Thanks for pointing this out.

**Reply by time4tea** [reply]
You can see the docs on GitHub - https://github.com/time4tea/gopro-dashboard-overlay/tree/main/docs

**Comment by falumas** [comment]
Just wanted to thank you for this awesome software. 

I use the Docker Container on Windows 11, Docker Desktop on WSL2 and has a Hero 8. Takes 2-3 min to process a 3.9GB (2.7k/60fps) GoPro Video.

**Comment by jchinte** [comment]
Hi! Thanks for all the hard work on this! I was looking around for a while for a way to do this.

My video was filmed upside down, so the overlay is placed upside down on top of the video. I used this hack in `ffmpeg.py` to solve my issue, as the ffmpeg options would not work for me:

`"-filter_complex", f"[0:v]hflip,vflip [bg];[bg][1:v]overlay{filter_extra}",`

It flips the video upside down before adding the overlay. The orientation is somehow still in the metadata so it plays back normally in windows media player.

Is an option in the works for this issue? Thanks so much again!

**Reply by time4tea** [reply]
Thanks for this. If the entire video is filmed upside down, you usually don't have to do anything...my gopro was mounted upside down for ages. I think perhaps there might be a case where the recording starts the right way up, then camera is immediately turned upside down?
Thanks for this info though - very useful!
